docs/fuzz-targets.md

```diff
@@ -0,0 +1,171 @@
+# Fuzzing using fuzz targets and the CLI
+
+Creating fuzz targets and executing those via CLI commands is straight forward
+and similar to what you would expect from other fuzzers. How to do so is
+described in detail in the following sections.
+
+## Setting up Jazzer.js
+
+Before you can use Jazzer.js you have to add the required dependency
+`@jazzer.js/core` to your project.
+
+```shell
+npm install --save-dev @jazzer.js/core
+```
+
+This will install Jazzer.js and all required dependencies in your project.
+
+## Creating a fuzz target
+
+Jazzer.js requires an entry point for the fuzzer, this is commonly referred to
+as fuzz target. A simple example is shown below.
+
+```js
+module.exports.fuzz = function (data) {
+	myAwesomeCode(data.toString());
+};
+```
+
+A fuzz target module needs to export a function called `fuzz`, which takes a
+`Buffer` parameter and executes the actual code under test.
+
+The `Buffer`, a subclass of `Uint8Array`, can be used to create needed
+parameters for the actual code under test, so that the fuzzer can detect the
```

Comment:
IMO the part with the detection sounds a bit confusing.

Reply:
True, removed. It will be added to the "value profiling" section.